lib/helmsman/helm.rb

```diff
@@ -32,11 +33,36 @@ def li_content
       disabled? ? link : "#{link}#{additional}".html_safe
     end
 
+    def configure_li_class
+      if @li_class.respond_to?(:to_ary)
+        @li_class.to_ary
+      else
+        @li_class.to_s.split(/\s+/)
+      end.tap do |li_class|
+        yield li_class
+      end.select(&:present?).uniq.sort * ' '
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.